src/views/HomeView.vue

```diff
@@ -39,29 +39,41 @@
               variant="outlined"
               color="primary"
               class="mt-4"
-              :small="$vuetify.display.mdAndDown ? true : false"
               to="/rules"
               data-cy="rules-link"
+              v-bind="{
```

Comment:
These were a bit trickier, we need to bind the prop since they're no longer booleans.

Reply:
Yeah funky one. What do you think of swapping the ternary into the value instead of the whole object?